R/AllGenerics.R

```diff
@@ -104,3 +104,4 @@ setGeneric("isolationWindowUpperMz", function(object, ...)
     standardGeneric("isolationWindowUpperMz"))
 setGeneric("filterPrecursorMz", function(object, ...)
     standardGeneric("filterPrecursorMz"))
+setGeneric("align", function(x, y, ...) standardGeneric("align"))
```

Comment:
1. This should probably be added to `ProtGenerics`
2. What about a less generic name that *align*? Something like `alignRt()` to avoid conflicts with totally unrelated fields?

Reply:
Totally agree. So, let's go for `alignRt` then. I will make a PR against `ProtGenerics` with that method.

Reply:
PR: https://github.com/lgatto/ProtGenerics/pull/13

Reply:
Commit https://github.com/lgatto/MSnbase/pull/535/commits/a23c3b11bdc0ca106469351a8e83de40faf36b3b renames `align` to `alignRt` and imports the generic method from `ProtGenerics`.